extensions/ql-vscode/src/view/model-editor/MethodRow.tsx

```diff
@@ -203,6 +221,21 @@ const ModelableMethodRow = forwardRef<HTMLElement | undefined, MethodRowProps>(
                 />
               ))}
             </MultiModelColumn>
+            {viewState.showMultipleModels && (
+              <MultiModelColumn gridColumn={6}>
+                {modeledMethods.map((_, index) => (
+                  <CodiconRow key={index}>
```

Comment:
It might be possible to merge the `CodiconRow` and `VSCodeButton` elements into one by putting the custom styling on the button component.

Reply:
Ok, I've done this. It has changed the styling slightly so the button is now the full height of a row instead of being centrally spaced within the row. But I still think it looks fine. I've updated the screenshot in the main description.